src/get-logger.js

```diff
@@ -50,6 +52,16 @@ function getShouldLogFn(...acceptableValues) {
   acceptableValues = ['', 'debug', ...acceptableValues]
```

Comment:
I think it's preferable to treat params (`acceptableValues `) as const and not to override them, to avoid any kind of confusion in readability